libraries/card-input/src/main/java/com/trendyol/cardinput/CardInputViewState.kt

```diff
@@ -0,0 +1,98 @@
+package com.trendyol.cardinput
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.drawable.Drawable
+import androidx.annotation.ColorInt
+import androidx.annotation.DrawableRes
+
+data class CardInputViewState(
+    val cardNumberTitle: String,
+    val expiryTitle: String,
+    val cvvTitle: String,
+    private val expiryMonthTitle: String,
+    private val expiryYearTitle: String,
+    val validationEnabled: Boolean = false,
+    val showCvvInfoButton: Boolean = true,
+    @DrawableRes private val inputBackground: Int = R.drawable.shape_card_input_field_background,
```

Comment:
```suggestion
    @DrawableRes private val inputBackgroundResouce: Int = R.drawable.shape_card_input_field_background,
```
Please add the "resource" end of an object name to show what the object is. Is it a drawable resource or drawable?